contracts/RegistrarController.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: UNLICENSED
+
+import "./interfaces/IRegistrarController.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+
+pragma solidity ^0.8;
```

Comment:
// line 6 
// pragma solidity 0.8.24;
// as best practice, you should use the latest version of solidity
// - 1 point